### PR TITLE
Add control file for OCP 1.3.0

### PR DESCRIPTION
**Pull request by rhmdnd**
This commit uses a script still under  (see https://github.com/ComplianceAsCode/content/pull/10455) to generate a control
file for the OpenShift CIS 1.3.0 benchmark.

The control file isn't complete, but it does provide the boilerplate
control structure based on the benchmark.

This makes it easier for contributors to come through and fill in the
missing pieces, like rules and status.

### CODE REVIEW

File: controls/cis_ocp_1.3.0.yml [hunk 1]
```diff
@@ -0,0 +1,757 @@
...
+  inherits_from: PLACEHOLDER
+controls:
+- id: '1'
+  title: Control Plane Components
+  status: PLACEHOLDER
+  rules: []
+  controls:
+  - id: '1.1'
+    title: Master Node Configuration Files
+    status: PLACEHOLDER
```
Comment (Mab879) [inline controls/cis_ocp_1.3.0.yml:19]: It might be better to use `planned` or `pending` for these statuses.

File: controls/cis_ocp_1.3.0.yml [hunk 2]
```diff
@@ -0,0 +1,757 @@
...
+    - id: 1.1.1
+      title: Ensure that the API server pod specification file permissions are set
+        to 600 or more restrictive
+      status: PLACEHOLDER
+      rules: []
+      level: level_1
+    - id: 1.1.2
+      title: Ensure that the API server pod specification file ownership is set to
+        root:root
+      status: PLACEHOLDER
```
Comment (rhmdnd) [inline controls/cis_ocp_1.3.0.yml:31]: This is causing CI to fail because it's not a valid status.

**Comment by rhmdnd** [comment]
> Just a minor comment on the status field, that is what is causing the build to fail. I would suggest looking into splitting file based on section (my idea would be to have `1.yml`, 2.yml` or similar) to help limit the size of this file.

Good idea. @jhrozek had similar feedback.

I proposed them as separate files, but each contains the same header information (metadata about the benchmark). Does this need to be removed?

**Comment by Mab879** [comment]
> > Just a minor comment on the status field, that is what is causing the build to fail. I would suggest looking into splitting file based on section (my idea would be to have `1.yml`, 2.yml` or similar) to help limit the size of this file.
> 
> Good idea. @jhrozek had similar feedback.
> 
> I proposed them as separate files, but each contains the same header information (metadata about the benchmark). Does this need to be removed?

Usually we only metadata in the main file so I would suggest moving it to the files clean. Behind the sense the build system just merges the files into one dict.

**Review by jhrozek — APPROVED** [review]
/lgtm
this works for me, any other changes we can iterate on

**Review by Mab879 — APPROVED** [review]
Its good a placeholder. Thanks for the PR!

**Pull request merged**